truss/getstarted/getstarted.go

```diff
@@ -0,0 +1,116 @@
+package getstarted
+
+import (
+	"bytes"
+	"os"
+	"strings"
+	"text/template"
+
+	log "github.com/Sirupsen/logrus"
+	gogen "github.com/golang/protobuf/protoc-gen-go/generator"
+	"github.com/pkg/errors"
+)
+
+type protoInfo struct {
+	alias string
+}
+
+func (p protoInfo) FileName() string {
+	return p.PackageName() + ".proto"
+}
+
+func (p protoInfo) PackageName() string {
+	a := p.alias
+	a = strings.Replace(a, "-", "", -1)
+	a = strings.Replace(a, "_", "", -1)
```

Comment:
There's nothing preventing them from having underscores; when implementing this I removed underscores due to some notion of aesthetics that was in my brain. I'll remove that if you think they should be allowed.